src/bin/bat/app.rs

```diff
@@ -80,6 +80,7 @@ impl App {
         let paging_mode = match self.matches.value_of("paging") {
             Some("always") => PagingMode::Always,
             Some("never") => PagingMode::Never,
+            // FIXME: `_` will always cover in or patterns
             Some("auto") | _ => {
```

Comment:
I don't know what to do about this. I could remove `Some("auto")` but it may change meaning of code.

Reply:
It would not change the meaning of the code. Maybe a better alternative would be:
``` rust
  Some("always") => PagingMode::Always,
  Some("never") => PagingMode::Never,
  None | Some("auto") => { … },
  _ => unreachable!(),
```

Reply:
Then `Some("*anything*")` will be reached on unreachable branch. Do you accept that?

Reply:
Yes. `Some("*anything*")` is `unreachable!()`. The argument parser enforces that:
``` rust
            Arg::with_name("paging")
                // ..
                .possible_values(&["auto", "never", "always"])
```